apps/jobbrowser/src/jobbrowser/templates/job_browser.mako

```diff
@@ -3079,6 +3098,10 @@ ${ commonheader("Job Browser", "jobbrowser", user, request) | n,unicode }
 
         lastFetchJobRequest = self._fetchJob(function (data) {
           if (data.status == 0) {
+            if (window.KNOX_BASE_URL.length && window.KNOX_BASE_URL.length) {
```

Comment:
length check twice, first one should probably be to check if window.KNOX_BASE_URL is defined?